WebServices/AzureNotificationHub/NotificationHubSample/NotificationHubSample.iOS/NotificationHubSample.iOS.csproj

```diff
@@ -55,7 +55,7 @@
     <WarningLevel>4</WarningLevel>
     <ConsolePause>false</ConsolePause>
     <MtouchArch>ARM64</MtouchArch>
-    <CodesignKey>iPhone Developer</CodesignKey>
+    <CodesignKey>iPhone Developer: Justin Johnson (7Z7BVUVAR3)</CodesignKey>
```

Comment:
You need to take your id out, as it'll just cause build issues for users.

Reply:
Good catch